templates/kube-controller-manager.yaml

```diff
@@ -18,6 +18,10 @@ spec:
     - --leader-elect=true
 {% if terminated_pod_gc_threshold is defined %}
     - --terminated-pod-gc-threshold={{terminated_pod_gc_threshold}}
+{% endif %}
+{% if kube_enable_cloud_provider %}
+    - --cloud-config={{kube_cloud_config_path}}
+    - --cloud-provider={{kube_cloud_provider}}
```

Comment:
@naveens I do not think this comes in the path of the out of tree cp you are referring to. This should be fine. Anyways this will be turned off when we use our CP today.